packages/node-bridge/launcher.js

```diff
@@ -132,6 +142,54 @@ function getVercelLauncher({
   };
 }
 
+/**
+ * @param {Function} listenerHandler
+ * @returns {(req: import('./types').VercelRequest, res: import('./types').VercelResponse) => void}
+ */
+function wrapListener(listenerHandler) {
+  // TODO: how can we best test the timeout and warning behavior?
+  return async function (req, res) {
+    enforceTimeout(res);
+    return await validateResponse(listenerHandler, req, res);
+  };
+}
+
+/**
+ * @param {Function} listenerHandler
+ * @param {import('./types').VercelRequest} req
+ * @param {import('./types').VercelResponse} res
+ */
+async function validateResponse(listenerHandler, req, res) {
+  const resp = await listenerHandler(req, res);
+  if (typeof resp !== 'undefined') {
+    // TODO: add vercel.link ?
+    // TODO: only show this once per invocation of this function?
```

Comment:
It could get really annoying if you wanted to return a value from your serverless function, for some reason. I don't know when you'd want this, though.

We could ship this and see if anyone complains.